client/views/msg_unhandled.tpl

```diff
@@ -5,7 +5,9 @@
 	<span class="from">[{{command}}]</span>
 	<span class="text">
 		{{#each params}}
-			<span>{{this}}</span>
+			<span>
+				{{this}}{{#equal ../command "421"}}{{#if @first}}:{{/if}}{{/equal}}
```

Comment:
Why the `../`? Also, could you add a comment maybe? It's very cryptic looking at this without an explication 😅